README.md

```diff
@@ -0,0 +1,31 @@
+Chameleon Base
+==============
+
+This is the base repository of the Chameleon System, providing core functionality. This repository should be installed
+and used in conjunction with the [Chameleon System](https://github.com/chameleon-system/chameleon-system). See that
+repository for details.
+
+Documentation
+-------------
+
+Please see the [Chameleon System main repository](https://github.com/chameleon-system/chameleon-system) for details on
+documentation.
+
+Installation
+------------
+
+Please see the [Chameleon System main repository](https://github.com/chameleon-system/chameleon-system) for an
+installation guide.
+
+License
+-------
+
+This repository is licensed under the MIT License. See the complete license under `LICENSE` for details.
+This repository also includes works under other copyright notices. See `THIRD-PARTY-LICENSE` for details.
+
+Reporting an Issue or a Feature Request
+---------------------------------------
+
+Please report issues and feature requests at the official GitHub repository of the [Chameleon System main
```

Comment:
```suggestion
Please report issues and feature requests at the chameleon-system repository of the [Chameleon System main
```

Reply:
I think the proposal would lead to confusion, as the namespace of all repos is also chameleon-system. The reader only needs to click the provided link, so I think the original sentence is OK.

Reply:
But still this one here is at least "Github" already?

Reply:
How about main repository?

Reply:
The sentence is now simplified - I think it still covers the essential information.